cli/args/mod.rs

```diff
@@ -651,6 +652,15 @@ impl CliOptions {
     let maybe_vendor_folder =
       resolve_vendor_folder(&initial_cwd, &flags, maybe_config_file.as_ref());
 
+    if let Some(env_file_name) = &flags.env_file {
+      from_filename(env_file_name).unwrap_or_else(|_| {
+        panic!(
+          "Unable to load '{}' environment variable file",
+          env_file_name.as_str()
+        )
+      });
+    }
+
```

Comment:
This block looks wrong to me. How would we like to handle the error?

Reply:
Use `bail!` instead which will return an error